docs/LoaderLayerInterface.md

```diff
@@ -0,0 +1,1732 @@
+  * [Exception for Elevated Privileges](#exception-for-elevated-privileges)
+* [Layer Version Negotiation](#layer-version-negotiation)
+* [Layer Call Chains and Distributed Dispatch](#layer-call-chains-and-distributed-dispatch)
+* [Layer Unknown Physical Device Extensions](#layer-unknown-physical-device-extensions)
+* [Layer Intercept Requirements](#layer-intercept-requirements)
+* [Distributed Dispatching Requirements](#distributed-dispatching-requirements)
+* [Layer Conventions and Rules](#layer-conventions-and-rules)
+* [Layer Dispatch Initialization](#layer-dispatch-initialization)
+* [Example Code for CreateInstance](#example-code-for-createinstance)
+* [Example Code for CreateDevice](#example-code-for-createdevice)
+* [Meta-layers](#meta-layers)
+* [Pre-Instance Functions](#pre-instance-functions)
+* [Special Considerations](#special-considerations)
+  * [Associating Private Data with Vulkan Objects Within a Layer](#associating-private-data-with-vulkan-objects-within-a-layer)
+    * [Wrapping](#wrapping)
+    * [Hash Maps](#hash-maps)
+  * [Creating New Dispatchable Objects](#creating-new-dispatchable-objects)
+* [Layer Manifest File Format](#layer-manifest-file-format)
+  * [Layer Manifest File Version History](#layer-manifest-file-version-history)
+* [Layer Library Versions](#layer-library-versions)
+  * [Layer Library API Version 2](#layer-library-api-version-2)
+  * [Layer Library API Version 1](#layer-library-api-version-1)
+  * [Layer Library API Version 0](#layer-library-api-version-0)
+
+
+## Overview
+
+This is the Layer-centric view of working with the Vulkan loader.
+For the complete overview of all sections of the loader, please refer 
+to the [LoaderInterfaceArchitecture.md](LoaderInterfaceArchitecture.md) file.
+
+
+## Layer Discovery
+
+As mentioned in the
+[Implicit versus Explicit](LoaderApplicationInterface.md#implicit-vs-explicit-layers),
+section of the
+[LoaderApplicationInterface.md](LoaderApplicationInterface.md) document, layers
+can be categorized into two categories:
+ * Implicit Layers
+ * Explicit Layers
+
+The main difference between the two is that implicit layers are automatically
+enabled, unless overridden, and explicit layers must be enabled.
+Remember, implicit layers are not present on all Operating Systems (like
+Android).
+
+On any system, the loader looks in specific areas for information on the layers
+that it can load at a user's request.
+The process of finding the available layers on a system is known as Layer
+Discovery.
+During discovery, the loader determines what layers are available, the layer
+name, the layer version, and any extensions supported by the layer.
+This information is provided back to an application through
+`vkEnumerateInstanceLayerProperties`.
+
+The group of layers available to the loader is known as a layer library.
+This section defines an extensible interface to discover what layers are
+contained in the layer library.
```

Comment:
I don't agree, but I think it's because of the language used here.  A `Layer Library` is a term used to define all the layers available to the loader.  Not the .so/.dll library file.  I went back and modified  "layer library" in these cases to "`Layer Library`".  I also cleaned up some wording because it was downright confusing.

Reply:
After talking off-line, we came to modifying this to "The `Layer Library`" for clarity.